core/catalog-core-urlresourcereader/src/main/java/ddf/catalog/resource/impl/URLResourceReader.java

```diff
@@ -187,7 +184,6 @@ public ResourceResponse retrieveResource(URI resourceURI, Map<String, Serializab
 
         if (properties.containsKey(BYTES_TO_SKIP)) {
             bytesToSkip = properties.get(BYTES_TO_SKIP).toString();
-            logger.debug("bytesToSkip: {}", bytesToSkip);
```

Comment:
I would leave the logging of this in there although you might update it be a little more verbose.  But this looks like it could come in handy for debugging in seeing if it got a bytesToSkip property